src/main/java/uk/gov/register/serialization/RegisterResult.java

```diff
@@ -12,11 +12,18 @@
     private Boolean isSuccessful;
     private String message;
     private Optional<Exception> exception;
+    private String details;
+
+    public RegisterResult() {
+        super();
```

Comment:
`super()` isn't needed here as it's only necessary in constructors of classes that extend some other base class.